### PR TITLE
Basiacarvalho/feature/movie list body

### PR DESCRIPTION
**Pull request by basiacarvalho**
Przesyłam całą sekcję movie-list

### CODE REVIEW

File: src/sass/partials/_movie-list.scss
Comment (piotrlewoniec) [inline src/sass/partials/_movie-list.scss:1]: Please consider using @use '../utils/variables' as var; instead only  @use '../utils/variables' ; for future code re-use.
After file name changed you can make one change and also two of developers already using it so for keeping code standard. (najlepsze jest, że nie wiem czy pro o takie coś by zwrócił uwagę :D , wieć trudno How bad is my english ? XD )

File: src/sass/utils/_mediaquery.scss
Comment (piotrlewoniec) [inline src/sass/utils/_mediaquery.scss:1]: line 14 you added + ?

**Review by piotrlewoniec — CHANGES_REQUESTED** [review]
Please take a look for given comments

File: src/sass/partials/_movie-list.scss
Comment (basiacarvalho) [reply src/sass/partials/_movie-list.scss:1]: Ok, done :)

**Review by piotrlewoniec — APPROVED** [review]
OK

**Pull request merged**